samples/ChatSample/Hub/Chat.cs

```diff
@@ -13,10 +13,10 @@ public void BroadcastMessage(string name, string message)
             Clients.All.SendAsync("broadcastMessage", name, message);
         }
 
-        [Authorize]
+        //[Authorize]
```

Comment:
If require Authorize, echo not work.
Is it a bug in SDK?

Reply:
It is added for testing in the old days. It will fail intentionally. You can remove it now.
Also we should add test to cover this scenario in the automation test.